# Photometric reprojection Error
uint32 state_state
FrameHessianData frame_hessian
geometry_msgs/Vector3 centerProjectedTo